lib/scss/inc/_variables.scss

```diff
@@ -31,20 +30,28 @@ $link-color: #2d91e6;
 // Font
 $font-size-base: 18px;
 $font-family: 'Source Sans Pro', sans-serif;
-$font-weight-bold: 600;
+$font-weight-bold: 500;
```

Comment:
Source Sans Pro doesn't have a 500 weight available, so we only loaded 400 and 600 weights for it on `index.erb`. Therefore we would need to use 600 weight here instead of 500.